.pre-commit-config.yaml

```diff
@@ -8,7 +8,7 @@ repos:
     rev: "23.1.0"
     hooks:
       - id: black
-        language_version: python3.8
```

Comment:
I updated this since 3.8 is EOL now. Python 3.11 is the latest version supported by Black 23.1.0.